processor/src/host/advice/mod.rs

```diff
@@ -138,6 +138,14 @@ impl AdviceProvider {
         &self.stack
     }
 
+    /// Extends the stack with the given elements.
```

Comment:
Would it be useful to explain that these elements are added to the top of the stack, and the last element of the iterator is the first one popped?

Reply:
Yes, I think it's worth clarifying. This would be basically equivalent to calling `push_stack()` in a loop.